### PR TITLE
ADD: remove file mutation

### PR DESCRIPTION
**Pull request by madisonmay**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] Jira story/task is put into PR title
- [ ] Code has been self-reviewed
- [ ] Hard-to-understand areas have been commented
- [ ] Documentation has been updated
- [ ] My changes generate no new warnings
- [ ] Unit tests have been added for base functionality and known edge cases
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: indico/queries/datasets.py [hunk 1]
```diff
@@ -262,7 +262,27 @@ def requests(self):
                 debouncer.backoff()
         yield GetDataset(id=dataset_id)
 
+       
+class RemoveFile(GraphQLRequest):
+    query = """
```
Comment (goatrocks) [inline indico/queries/datasets.py:267]: Could you add pydoc comments to this, and run sphinx build if you're able? Otherwise just update the comments and I can run the build script. Otherwise, looks good! Thank you.

File: indico/queries/datasets.py [hunk 2]
```diff
@@ -262,7 +262,27 @@ def requests(self):
                 debouncer.backoff()
         yield GetDataset(id=dataset_id)
 
+       
+class RemoveFile(GraphQLRequest):
```
Comment (goatrocks) [inline indico/queries/datasets.py:266]: Could we call this RemoveDatasetFile/something to indicate it is removing from a dataset?

**Review by goatrocks — CHANGES_REQUESTED** [review]

**Comment by madisonmay** [comment]
Absolutely can make the requested changes. Thanks Meg!

File: indico/queries/datasets.py (hunk 2 above)
Comment (madisonmay) [reply indico/queries/datasets.py:266]: Yup! I went with the name for consistency with AddFiles, but what I can do is add an alias for AddFiles to match this naming convention

**Comment by madisonmay** [comment]
The codeship error seems unrelated also @goatrocks ?

**Comment by goatrocks** [comment]
> 

Yeah, the codeship error does seem unrelated -- we can re-run the build. Also, the large diff tends to happen with sphinx.... The only issue would be if your local master branch wasn't up to date when you created this branch, otherwise it should likely be fine. @madisonmay
